transports/http_admin.go

```diff
@@ -316,3 +317,53 @@ func (h *TransportHTTP) AdminGetSharedConfig(ctx context.Context) (*models.Share
 
 	return model, nil
 }
+
+// AdminGetContacts executes an HTTP POST request to search for contacts based on specified conditions, metadata, and query parameters.
+func (h *TransportHTTP) AdminGetContacts(ctx context.Context, conditions map[string]interface{}, metadata *models.Metadata, queryParams *QueryParams) ([]*models.Contact, ResponseError) {
+	jsonStr, err := json.Marshal(map[string]interface{}{
+		FieldConditions:  conditions,
+		FieldMetadata:    processMetadata(metadata),
+		FieldQueryParams: queryParams,
+	})
```

Comment:
It's for the future refactorings, but we should choose one direction,
Either, set both request and response should be "declared" in go-client and we shouldn't have dependency to the models package.
Or we should have both request and response build from the models package.

Because currently we're building structure of request in the "isolated" way, so it can be not aligned with the spv-wallet endpoint contract, but the response is deserialised into spv-wallet structs.

Reply:
We will address this, but after the refactoring process. 
I'm gonna introduce models for filters to solve this - but it's better to do it on refactored codebase.